### PR TITLE
Holopads have better examine() and are now constructable

### PR DESCRIPTION
**Pull request by ghost**
image

~~I hate the word holopad. It brings such utter disgust to me. It's a literal ripoff of the Star Wars franchise. Why not have a cooler name, like teleprojector? I hate holopads. Once, in OOC, Colfer talked about how he wanted holographic crew members I was just absolutely seething. Holopad code is atrocious, and before I die, I will replace them with teleprojectors.~~

A new type of constructible machine frame, the floor machine frame, has been added. It's not dense. That's all that's special about it, really. Holopads can now be constructed and deconstructed. They need a scanning module, microlaser, and input controller. Holopad circuits are now available for research. Holopads have a better `examine()` rather than a mediocre customized description. Regular machine frames have a description now. I've replaced much of the HTML tags with macros. Don't ask why. I did some sprite organization and cleanup.

### CODE REVIEW

File: code/game/machinery/hologram.dm [hunk 1]
```diff
@@ -115,17 +119,17 @@ var/const/HOLOPAD_MODE = RANGE_BASED
 				var/temppad = input(user, "Which holopad would you like to contact?", "holopad list") as null|anything in holopadlist
 				targetpad = holopadlist["[temppad]"]
 				if(targetpad==src)
-					to_chat(user, "<span class='info'>Using such sophisticated technology, just to talk to yourself seems a bit silly.</span>")
-					targetpad = null //Clean up the mess after an unsuccessful call
+					to_chat(user, SPAN_INFO("Using such sophisticated technology, just to talk to yourself seems a bit silly."))
+         	targetpad = null //Clean up the mess after an unsuccessful call
```
Comment (ghost) [inline code/game/machinery/hologram.dm]: Damn it.

**Comment by Anton-Kr** [comment]
Not sure how I feel about a one off construction thing for JUST holopads. Any issues with just updating the density on finishing constructions on a regular machine? Not a huge deal if you REALLY want to keep it. Further, any chance you can add long distance holopads to this as well?

**Comment by ghost** [comment]
> Not sure how I feel about a one off construction thing for JUST holopads. Any issues with just updating the density on finishing constructions on a regular machine? Not a huge deal if you REALLY want to keep it. Further, any chance you can add long distance holopads to this as well?

The floor machine frame can be adopted elsewhere as needed, ~~primarily vents that magically fucking appear out of pipe dispensers~~. I deem it cool and necessary for future floor machinery.

> Further, any chance you can add long distance holopads to this as well?

Long-range holopads are in this.

**Comment by Anton-Kr** [comment]
Fair. Mind giving long-range holopads an extra component, something from tcomms to simulate them being long range and all? 

>primary vents

oh god plz no

**Comment by ghost** [comment]
> Fair. Mind giving long-range holopads an extra component, something from tcomms to simulate them being long range and all?
> 

I'd rather not considering away-site people having to rebuild a holopad probably won't have access to telecommunications components. I accidentally maimed git aggressively and will need to re-open the PR in a couple minutes.

> > primary vents
> 
> oh god plz no

Coping initiated.

**Comment by ProbablyCarl** [comment]
Good shit.

**Review by Anton-Kr — APPROVED** [review]

**Pull request merged**